composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "php-http/httplug-bundle",
+    "description": "Symfony bundle to integrate the Httplug generic HTTP client into Symfony",
+    "type": "symfony-bundle",
+    "keywords": ["http", "discovery", "adapter", "message", "factory", "bundle", "httplug", "php-http"],
+    "homepage": "http://php-http.org",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "David Buchmann",
+            "email": "mail@davidbu.ch"
+        }
+    ],
+    "require": {
+        "php-http/discovery": "^0.1.1",
+        "php-http/httplug": "^0.1.0",
+        "php-http/httplug-implementation": "^0.1.0",
```

Comment:
but depending on php-http/psr-client-implementation should be enough,
right? if people need a client that has the convenience methods, they
can just add that to their dependencies and configure it, but psr-client
is kind of the minimum...